### ISSUE TITLE
Uncaught TypeError: Cannot read property 'validate' of undefined at HTMLSelectElement.handleChange

### ISSUE THREAD

**Issue by caschbre** [tjinauyeung/svelte-forms-lib#10]
**Describe the bug**
While attempting to run the helper components demo on a fresh svelte app, the following error appears:
`Uncaught TypeError: Cannot read property 'validate' of undefined at HTMLSelectElement.handleChange (createForm.js:52)`

**To Reproduce**
1. Create new svelte app (npx degit sveltejs/template svelte-app)
2. Install svelte-forms-lib (npm i -D ....)
3. Install yup (npm i -D yup)
4. Edit App.svelte and copy / paste from https://svelte-forms-lib-sapper-docs.now.sh/helpers
5. Change `import yup from "yup"` -> `import * as yup from "yup"`
6. Add Select to the import statements.
7. npm run dev
8. Open app in browser and then tab through the form elements.

**Expected behavior**
For the fresh svelte app to work like the demo without the error.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 18
 - Browser: Chrome
 - Version: 80

**Additional context**
The Field (https://svelte-forms-lib-sapper-docs.now.sh/field) demo doesn't show the error, however it also doesn't execute the onSubmit.

**Comment by caschbre** [comment]
I just tested our the yup demo  (https://svelte-forms-lib-sapper-docs.now.sh/yup) and that does appear to work without issue.  So may be the issue is the helper components?

**Comment by tjinauyeung** [comment]
Found the issue. The example had an incomplete `validationSchema`. Not taking into account the `title` for the select, which was added later.

I've updated the docs. Thanks for finding out.

Another note; when installing with the `-D` you're installing it to the `devDependencies` which should only contain dependencies used for developing such as tooling. Both `svelte-forms-lib` and `yup` should be installed in as normal dependencies
